packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -75,8 +75,8 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
 
   static const summaryTreeKey = Key('Summary Tree');
   static const detailsTreeKey = Key('Details Tree');
-  static const includeTextWidth = 900.0;
-  static const includeRefreshTreeWidth = 1255.0;
+  static const unscaledIncludeTextWidth = 900.0;
```

Comment:
most of the time the width values we pass into widgets are all real pixels. For this case, there are some classes that are nicer to keep const so we pass in the unscaled version of the width instead.